worlds/sc2/test/test_items.py

```diff
@@ -0,0 +1,38 @@
+import unittest
+from typing import List
+
+from worlds.sc2 import items
+
+
+class TestItems(unittest.TestCase):
+    def test_grouped_upgrades_number(self) -> None:
+        bundled_items = items.upgrade_bundles.keys()
+        bundled_item_data = [items.get_full_item_list()[item_name] for item_name in bundled_items]
+        bundled_item_numbers = [item_data.number for item_data in bundled_item_data]
+
+        check_numbers = [True if number == -1 else False for number in bundled_item_numbers]
+
+        self.assertNotIn(False, check_numbers)
+
+    def test_non_grouped_upgrades_number(self) -> None:
+        check_modulo = 4
+        bundled_items = items.upgrade_bundles.keys()
+        non_bundled_upgrades = \
+            [
```

Comment:
Inconsistent style -- AP style docs and our other code says to start the collection on the same line as the `=`:
```python
non_bundled_upgrades = [
    # items
]
```